taskana-adapter-taskana-connector/src/main/java/pro/taskana/adapter/taskanaconnector/api/impl/TaskInformationMapper.java

```diff
@@ -110,6 +112,33 @@ public ReferencedTask convertToReferencedTask(Task taskanaTask) {
     return referencedTask;
   }
 
+  private void setCustomIntegers(ReferencedTask referencedTask, TaskImpl taskanaTask) {
+    if (referencedTask.getCustomInt1() != null && !referencedTask.getCustomInt1().isEmpty()) {
+      taskanaTask.setCustomInt1(Integer.parseInt(referencedTask.getCustomInt1()));
+    }
+    if (referencedTask.getCustomInt2() != null && !referencedTask.getCustomInt2().isEmpty()) {
+      taskanaTask.setCustomInt2(Integer.parseInt(referencedTask.getCustomInt2()));
+    }
+    if (referencedTask.getCustomInt3() != null && !referencedTask.getCustomInt3().isEmpty()) {
+      taskanaTask.setCustomInt3(Integer.parseInt(referencedTask.getCustomInt3()));
+    }
+    if (referencedTask.getCustomInt4() != null && !referencedTask.getCustomInt4().isEmpty()) {
+      taskanaTask.setCustomInt4(Integer.parseInt(referencedTask.getCustomInt4()));
+    }
+    if (referencedTask.getCustomInt5() != null && !referencedTask.getCustomInt5().isEmpty()) {
+      taskanaTask.setCustomInt5(Integer.parseInt(referencedTask.getCustomInt5()));
+    }
+    if (referencedTask.getCustomInt6() != null && !referencedTask.getCustomInt6().isEmpty()) {
+      taskanaTask.setCustomInt6(Integer.parseInt(referencedTask.getCustomInt6()));
+    }
+    if (referencedTask.getCustomInt7() != null && !referencedTask.getCustomInt7().isEmpty()) {
+      taskanaTask.setCustomInt7(Integer.parseInt(referencedTask.getCustomInt7()));
+    }
+    if (referencedTask.getCustomInt8() != null && !referencedTask.getCustomInt8().isEmpty()) {
+      taskanaTask.setCustomInt8(Integer.parseInt(referencedTask.getCustomInt8()));
```

Comment:
@gitgoodjhe should we catch the exception if `customX` is not a number?
What happens if the NumberFormatException is thrown here?